swift/src/IceImpl/Communicator.mm

```diff
@@ -1,15 +1,15 @@
 // Copyright (c) ZeroC, Inc.
-#import "Communicator.h"
-#import "DispatchAdapter.h"
-#import "IceUtil.h"
-#import "ImplicitContext.h"
-#import "Logger.h"
-#import "ObjectAdapter.h"
-#import "ObjectPrx.h"
-#import "Process.h"
-#import "Properties.h"
-#import "PropertiesAdmin.h"
-#import "UnsupportedAdminFacet.h"
+#import "include/Communicator.h"
```

Comment:
What goes into include vs what stays?

It's not immediately clear to me. And should this be documented in a README?